modules/nf-core/decoupler/progeny/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+process {
+    withName: "DECOUPLER_PROGENY" {
+        ext.prefix = 'test'
+        ext.args = { """{"organism": "human", "top_genes": 1000}""" }
```

Comment:
Could you supply these through the test, as described in the [guidelines](https://nf-co.re/docs/tutorials/tests_and_test_data/nf-test_writing_tests#nf-test-guidelines-for-a-simple-un-chained-module) please?